### PR TITLE
Dashboard Component

### PR DESCRIPTION
**Pull request by richarddubay**
# Feature / Fixed Issue(s)
- Adding the Dashboard component

# Testing/Documentation
- [x] All significant new logic is covered by tests.
- [x] Storybook documentation has been added.

# Screenshots
- If applicable, include screenshots of UI/style changes.

### CODE REVIEW

**Comment by jbaxleyiii** [comment]
**[Storybook Hub](https://hub.getstorybook.io/)** is building storybooks for every commit in this PR.

**Latest Storybook: https://54652110-0f14-4a8e-9ea3-b6fb22861204.sbook.io/**
<sub>( Visit [here](https://hub.getstorybook.io/app/github.com/NewSpring/Holtzman/branch/dashboard-component/) for more info )</sub>

> _This comment is automatically added by Storybook Hub_

**Comment by richarddubay** [comment]
@jbaxleyiii Thoughts so far? Feeling close? What am I missing?

**Comment by jbaxleyiii** [comment]
@richarddubay gah that came together so well (and quickly!!). A couple minor tweaks and then this is ready to be plumbed into the app!

- [ ] make the <h2> an <h1> (this will help with some SEO and generally help weight the page nicely)
- [ ] let's add some padding around the upper block (it will need a wrapper div to contain the upper block vs where the {children} goes). `soft-half soft@lap-and-up soft-double-top@lap-wide-and-up` I think that will get us the right amount! No padding / margin needed for the {children} part.

SO great!!!

File: imports/components/dashboard/index.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  children?: React$Element<any>,
+};
+
+const Dashboard = ({
+  title,
+  subNav,
+  children,
+}: IDashboard) => (
+  <div>
+    <h2 className="soft-bottom">{title}</h2>
```
Comment (jbaxleyiii) [inline imports/components/dashboard/index.js]: Let's make a wrapper around these elements (the `h2` and subnav) that we can set padding on and make the background--light-primary`

File: imports/components/dashboard/index.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    <div className="floating text-left">
+      {getLinks(subNav)}
+    </div>
+    <div className="background--light-secondary" style={{ borderTop: "1px solid #dddddd" }}>
```
Comment (jbaxleyiii) [inline imports/components/dashboard/index.js]: I *think* `outlined--top outlined--light` should do this same thing? If not, the inline styles work great!

File: imports/components/dashboard/index.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+      <div className="soft push">
```
Comment (jbaxleyiii) [inline imports/components/dashboard/index.js]: lets not add any shaping around the bottom content. That will allow us to do full bleed images / change the background, etc as needed

File: imports/components/dashboard/index.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    >
+      <h6>{x.title}</h6>
+    </Link>
+  ));
+  return links;
+};
+
+type IDashboard = {
+  title: string,
+  subNav: Object[],
```
Comment (jbaxleyiii) [inline imports/components/dashboard/index.js]: can we type the subnav as well?

File: imports/components/dashboard/index.js (hunk 1 above)
Comment (jbaxleyiii) [inline imports/components/dashboard/index.js]: also, let's make this a `h1`

**Pull request merged**